cluster/conf/debug.conf.template

```diff
@@ -1,3 +1,3 @@
-MEMBERS_FILE=members.txt
+MEMBERS_FILE=../work/members.txt
```

Comment:
Shouldn't the default be "$SPARK_HOME/work/members.txt"? Since the script just sources the conf file, the variable should expand just fine. Otherwise "../work" is dependent on the current directory of the user which can be somewhere else.